src/Components/Input.tsx

```diff
@@ -9,37 +9,34 @@ export interface InputProps extends React.HTMLProps<HTMLInputElement> {
   block?: boolean
   leftView?: JSX.Element
   rightView?: JSX.Element
+  value?: string
 }
 
 interface InputState {
   borderClasses: string
 }
 
 const StyledInput = styled.input`
-  ${borderedInput}
-  ${block(24)}
+  ${borderedInput} ${block(24)};
```

Comment:
Add a semi-colon after the interpolated value so prettier doesn’t move these both to a single line.

Reply:
I inherited this lol. I only intended to add `value` to the input props and prettier did its thing.

Reply:
Yeah I know :( It’s a PITA with prettier 1.5.x